blank-canvas-blocks/experimental-theme.json

```diff
@@ -175,44 +183,44 @@
 		},
 		"core/heading/h1": {
 			"typography": {
-				"fontSize": "var(--wp--preset--font-size--huge)",
-				"lineHeight": "var(--wp--custom--line-height--headings)"
+				"fontSize": "48px",
+				"lineHeight": "var(--wp--custom--heading--typography--line-height)"
 			}
 		},
 		"core/heading/h2": {
 			"typography": {
-				"fontSize": "var(--wp--preset--font-size--huge)",
-				"lineHeight": "var(--wp--custom--line-height--headings)"
+				"fontSize": "32px",
```

Comment:
why are we not using variables for h1 and h2 like we are doing for the rest?

Reply:
Because those sizes exceeded what was configured as font size. I want sure if it was a good idea to add two more sizes, which seem more focused on paragraph styles, just to style the larger header elements.

Reply:
I lean towards not adding more variables for headings, since we have the ability to change it directly in the global styles supported style properties of heading blocks.  

Unless these font sizes would be used again outside the H1 + H2. Now that I think about it, they might for a block like site title... Hmm.

Reply:
That's more of a decision at the "theme" level then the "polyfill" level and those additional variables certainly could be added by that theme (or this one if we decide to style the block title in that way).
`Don't code it 'till ya need it.`